conformance/conformance.proto

```diff
@@ -57,11 +57,13 @@ option java_package = "com.google.protobuf.conformance";
 //   2. parse the protobuf or JSON payload in "payload" (which may fail)
 //   3. if the parse succeeded, serialize the message in the requested format.
 message ConformanceRequest {
+  string test_name = 1;
```

Comment:
I guess I lean towards providing more information here for convenience, but we can take it out for now and add it on an as-needed basis.

One case where the test runner might not be able to log the failure is if the tests are running in-process (ie using the ConformanceTestSuite class directly). This is what the Objective C tests are planning to do for iOS. The runner can't print log messages during the test because iOS doesn't have stdin/stderr, so the runner is just accumulating the test output in a string. If the process crashes, the runner won't be able to print any output (even if `--verbose` was specified).